src/nerve/frontends/tui/commander/workflow_runner.py

```
@@ -0,0 +1,1028 @@
+            for s in self.steps
+        ]
+
+    async def run(self) -> dict[str, Any]:
+        """Run the workflow and return result."""
+        # Start the workflow
+        try:
+            result = await self.adapter.execute_workflow(self.workflow_id, self.input_text)
+            self.run_id = result.get("run_id", "unknown")
+            self.state = result.get("state", "running")
+            self._add_event("workflow_started", {"run_id": self.run_id})
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)